ext/node/polyfills/internal/crypto/cipher.ts

```diff
@@ -124,21 +117,28 @@ export interface DecipherOCB extends Decipher {
 }
 
 export class Cipheriv extends Transform implements Cipher {
+  /** CipherContext resource id */
+  context: number;
+
+  _cache: BlockModeCache;
+  _finished: false;
```

Comment:
Convert to private fields `#context`. It's consistent with the Hash class resource ids in node:crypto

Reply:
changed them to private